docs/articles/documentation/extending-testcafe/custom-reporter/index.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: docs
+title: Custom Reporter Plugin
+permalink: /documentation/extending-testcafe/custom-reporter-plugin/
+---
+# Custom Reporter Plugin
+
+TestCafe has a number of [built-in reporters](../../../using-testcafe/common-concepts/reporters.md) to generate test reports in different formats.
+You can also create a **custom reporter** that will output information in your own format and style.
+For this purpose, you can use the [TestCafe reporter generator](https://github.com/DevExpress/generator-testcafe-reporter).
+The generator will scaffold out a reporter plugin, so that you only need to write a few lines of code.
+
+For more information on how to implement the plugin, refer to [Creating Reporter Plugin](creating-reporter-plugin.md).
+
+When creating the plugin, you will need to use some helpers to format the report. You can find the reference in the [Helper Methods](helper-methods.md) topic.
```

Comment:
I like the current structure with separate topics. @VasilyStrelyaev, what do you think?